src/views/Dogs/Dogs.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { useState, useEffect } from 'react';
+import { fetchDogs } from '../../services/dog';
+import DogList from '../../components/DogList/DogList';
+import './Dogs.css';
+
+export default function Dogs() {
+  const [dogs, setDogs] = useState([]);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    let timer;
+    const getDogs = async () => {
+      const resp = await fetchDogs();
+      setDogs(resp);
+      timer = setTimeout(() => {
+        setLoading(false);
+      }, 750);
+    };
+    if (loading) {
```

Comment:
This `if(loading)` is not necessary . This useEffect should fetch the data once when the component mounts.